src/redux/books/books.js

```diff
@@ -1,13 +1,13 @@
-const ADDBOOK = 'ADDBOOK';
-const REMOVEBOOK = 'REMOVEBOOK';
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] As stated by the previous reviewer, kindly consider the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:       const ADD_BOOK = 'bookstore/books/ADD_BOOK'